src/openrct2-ui/windows/Ride.cpp

```diff
@@ -3130,7 +3134,7 @@ static void window_ride_vehicle_invalidate(rct_window* w)
         window_ride_vehicle_widgets[WIDX_VEHICLE_CARS_PER_TRAIN_DECREASE].type = WWT_EMPTY;
     }
 
-    auto ft = Formatter::Common();
+    ft = Formatter::Common();
```

Comment:
Rewind. It would be better if the ride format was just moved to after all of this then you don't need the increment 20